### DISCUSSION TITLE
else 3 handle HTTP status codes in SendHttpRequest?

### DISCUSSION THREAD

**Discussion by mustafaatatuzun** [elsa-workflows/elsa-core#4070] [Q&A]
how do we add at elsa 3  , the status code 200 ? the tag section is missing

**Comment by sfmskywalker** [comment]
Like this:

<img width="1715" alt="image" src="https://github.com/elsa-workflows/elsa-core/assets/938393/a355c91a-cf7f-4901-b371-83dad919d9cc">

**Reply by sfmskywalker** [reply]
I wanted to also show it for the **HTTP Request** activity, but now I see what you are referring to: the Expected Status Codes field seems to be missing there. I'll fix it.

**Reply by sfmskywalker** [reply]
Should be good now as of 68b36ec39efadb179e9e3c2b144f237b648ab292
